src/OnlineStatsBase.jl

```diff
@@ -72,7 +72,7 @@ Base.copy(o::OnlineStat) = deepcopy(o)
 """
     merge!(stat1, stat2)
 
-Merge `stat1` into `stat2` (supported by most `OnlineStat` types).
+Merge `stat2` into `stat1` (supported by most `OnlineStat` types).
```

Comment:
@joshday Seems like it should be described this way as stat1 is the one being mutated.